cogent/Makefile

```diff
@@ -64,25 +60,19 @@ doc:
 
 clean:
 	$(E) "Cleaning"
+	$(Q) $(CABAL) clean
+	$(Q) make -C doc clean
+	$(Q) find . -name "*_flymake.hs" -delete
+
+full-clean: clean
 	$(Q) $(CABAL) sandbox delete -v0
 	$(Q) rm -rf .cabal-sandbox/ dist/ ../isa-parser/dist
 	$(Q) rm -rf out/
```

Comment:
maybe we can clean everything minus sandbox in the `clean` target. The only thing I want from `clean` is to keep my dependencies.